packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## 46.4.0 - July 3, 2019
+
+### Fixed
+
+#### salesforcedx-vscode-apex
+
+- Fix `Declare missing method` quick fix option crashing when generating a method with nested return type ([PR #1444](https://github.com/forcedotcom/salesforcedx-vscode/pull/1444))
+
+#### salesforcedx-vscode-core
+
+- Improve deploy on save performance and optimize how subsequent saves are queued for deployment ([PR #1442](https://github.com/forcedotcom/salesforcedx-vscode/pull/1442), [Issue #1155](https://github.com/forcedotcom/salesforcedx-vscode/issues/1155))
```

Comment:
Let's word this differently because it gives the impression we fixed deploy performance and we only fixed how long it takes to trigger the deploy on save functionality. The deploy performance story will be coming soon so that's why I want to make it clear. cc @hbc-bindu

Reply:
Does this sound ok:
Update deploy on save behavior to run one deployment at a time, optimize how subsequent saves are queued for deployment, and trigger immediate deployment if there isn’t an active one.

Reply:
Yes, thanks for the update